### PR TITLE
refactor(floatbar): [floatbar] refactor theme vars

### PR DESCRIPTION
**Pull request by Youyou-smiles**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Updated the `<tiny-floatbar>` component to display list item labels in Chinese.

- **Bug Fixes**
	- Adjusted the hover effect and background colors for improved visual consistency.

- **Refactor**
	- Enhanced CSS variable management for the float bar component, including updated variable naming conventions and method for injecting CSS variables.

- **Tests**
	- Updated test cases to reflect changes in the float bar's styling and localization.

### CODE REVIEW

File: packages/theme/src/floatbar/index.less [hunk 1]
```diff
@@ -17,44 +17,43 @@
...
+      min-height: 32px;
+      line-height: 32px;
       text-align: center;
-      margin: var(--ti-float-bar-li-margin-vertical) var(--ti-float-bar-li-margin-horizontal);
+      margin: var(--tv-Float-bar-li-margin-vertical) 0;
+      padding: 0 24px;
 
       &:hover {
-        background: var(--ti-float-bar-list-hover-bg-color);
+        background: rgba(0, 0, 0, 0.05)
```
Comment (zzcr) [inline packages/theme/src/floatbar/index.less]: 这个颜色值应该需要使用变量

File: packages/theme/src/floatbar/index.less [hunk 2]
```diff
@@ -17,44 +17,43 @@
...
     li {
       display: block;
-      color: var(--ti-float-bar-list-text-color);
-      font-size: var(--ti-float-bar-font-size);
-      min-width: var(--ti-float-bar-list-item-min-width);
-      min-height: var(--ti-float-bar-list-item-min-height);
-      line-height: var(--ti-float-bar-list-line-height);
+      color: var(--tv-Float-bar-list-text-color);
+      font-size: var(--tv-Float-bar-font-size);
+      min-width: 014px;
```
Comment (zzcr) [inline packages/theme/src/floatbar/index.less]: 多了个0

File: packages/theme/src/floatbar/vars.less [hunk 3]
```diff
@@ -10,33 +10,19 @@
...
 */
 
-.component-css-vars-floatbar() {
-  // 浮动块边框色
-  --ti-float-bar-border-color: unset;
+.inject-Floatbar-vars() {
+
   // 浮动块圆角
-  --ti-float-bar-border-radius: var(--ti-common-border-radius-4, 8px);
+  --tv-Float-bar-border-radius: var(--tv-border-radius-lg);
```
Comment (zzcr) [inline packages/theme/src/floatbar/vars.less]: 组件名字应该叫：Floatbar

Comment (Youyou-smiles) [reply packages/theme/src/floatbar/vars.less]: > 组件名字应该叫：Floatbar

是的，立刻修改

File: packages/theme/src/floatbar/index.less (hunk 2 above)
Comment (Youyou-smiles) [reply packages/theme/src/floatbar/index.less]: > 多了个0

多谢提醒

File: packages/theme/src/floatbar/index.less [hunk 4]
```diff
@@ -17,44 +17,43 @@
...
+      min-height: 32px;
+      line-height: 32px;
       text-align: center;
-      margin: var(--ti-float-bar-li-margin-vertical) var(--ti-float-bar-li-margin-horizontal);
+      margin: var(--tv-Floatbar-bar-li-margin-v) 0;
+      padding: 0 24px;
 
       &:hover {
-        background: var(--ti-float-bar-list-hover-bg-color);
+        background: #ededed
```
Comment (zzcr) [inline packages/theme/src/floatbar/index.less:44]: 这个颜色值也应该抽成变量

Comment (Youyou-smiles) [reply packages/theme/src/floatbar/index.less:44]: vars.less文件中没有这个颜色的变量。

**Review by zzcr — APPROVED** [review]

**Pull request merged**